content/applications/inventory_and_mrp/quality/quality_management/quality_control_points.rst

```diff
@@ -1,89 +1,123 @@
-   :align: center
-   :alt: An example of a completed quality control point form for a Pass-Fail test.
-
-Once a control point has been configured, a quality check will be automatically created and
-assigned when the specified operation or work order has been reached. Quality checks can be managed
-by selecting :menuselection:`Quality Control --> Quality Checks`.
-
-.. tip::
-   To see all of the quality checks created by a control point, go to :menuselection:`Quality
-   Control --> Control Points`, select a control point, and click :guilabel:`Quality Checks` in the
-   top right corner.
-
-Use case: configure a measure quality check
-===========================================
-
-To ensure that a product meets specific measurement requirements, select :guilabel:`Measure` from
-the :guilabel:`Type` drop-down menu. Selecting the :guilabel:`Measure` quality check type reveals
-three new fields: :guilabel:`Device`, :guilabel:`Norm`, and :guilabel:`Tolerance`. These fields can
-be configured so that only products within a certain tolerance will pass the check:
-
-- :guilabel:`Device`: select the measuring device that should be used to take the measurement
-  (e.g., measuring tape).
-- :guilabel:`Norm`: specify the desired measurement that the product should conform to and the unit
-  of measurement that should be used
-- :guilabel:`Tolerance`: select the range that a measurement can be within while still passing the
-  check (e.g., :guilabel:`from` 59.5 mm :guilabel:`to` 60.5 mm)
-
-.. image:: quality_control_points/measure_quality_check.png
+.. |MO| replace:: :abbr:`MO (Manufacturing Order)`
+.. |MOs| replace:: :abbr:`MOs (Manufacturing Orders)`
+.. |QCP| replace:: :abbr:`QCP (Quality Control Point)`
+.. |QCPs| replace:: :abbr:`QCPs (Quality Control Points)`
+
+In Odoo, quality control points (QCPs), are used to automatically create :doc:`quality checks <quality_checks>`
+at predetermined intervals. |QCPs| can be configured to create quality checks for specific
+operations (manufacturing, delivery, etc.), as well as specific products within those operations.
+
+Using |QCPs| allows quality teams to ensure that products are being regularly inspected for defects
+and other issues.
+
+Configure a quality control point
+=================================
+
+To create a new |QCP|, navigate to :menuselection:`Quality --> Quality Control --> Control Points`,
+and then click :guilabel:`New`.
+
+Begin filling out the new |QCP| by entering a unique :guilabel:`Title` that will make the |QCP|
+easily identifiable.
+
+In the :guilabel:`Products` field, select one or more products that the |QCP| should apply to. If
+the |QCP| should apply to an entire product category, select it in the :guilabel:`Product
+Categories` field.
+
+In the :guilabel:`Operations` field, select the operation(s) that will trigger the |QCP|. For
+example, selecting the *Manufacturing* operation will cause a quality check to be created for new
+|MOs|.
```

Comment:
If you're referring to the manufacturing option the Operations dropdown, I'd use a guilabel here (:

Also, I believe this is the first time we used the abbr, MO, so we should define it!
```suggestion
In the :guilabel:`Operations` field, select the operation(s) that triggers the |QCP|. For example,
selecting the :guilabel:`Manufacturing` option will trigger a quality check for new manufacturing
orders (MO).
```